addon/src/modifiers/sortable-item.js

```diff
@@ -706,7 +824,8 @@ export default class SortableItemModifier extends Modifier {
    @type Number
    */
   get transitionDuration() {
-    let el = this.element;
+    const items = this.sortableGroup.sortedItems.filter((x) => !x.isDragging && !x.isDropping);
+    let el = items[0].element ?? this.element; // Fallback when only one element is present in list
```

Comment:
This was since now not correct... when someone has set transition to 1 sec or more, vertical and also horiziontal elements were jumping, because the duration of moved element is always "0"

Reply:
this change was moved to PR #562